script.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const path = require('path');
+
+const file = process.argv.slice(2)[0];
+
+const filePath = path.resolve(path.join(__dirname, file));
```

Comment:
no need for `path.resolve` here, since `__dirname` is absolute and so the result of `path.join` will be as well